code/controllers/subsystem/spatial_gridmap.dm

```diff
@@ -466,30 +582,43 @@ SUBSYSTEM_DEF(spatial_grid)
 /atom/proc/find_grid_statistics_for_z_level(insert_clients = 0)
```

Comment:
lol this proc does nothing but make it more annoying to add new contents

Reply:
should probably be removed or redone.

Reply:
I mean it's at least at barebones useful for checking if everything's working right.
You want me to nuke it?

Reply:
nah it can stay, i think it can work for every new contents type without needing to be added to if i store contents vars